src/KM_FormMain.dfm

```diff
@@ -40,19 +40,24 @@ object FormMain: TFormMain
         Width = 110
       end
       item
-        Text = 'Cursor: 46.1 47.2'
-        Width = 160
+        Text = 'Cursor: '
```

Comment:
Let's leave some placeholder numbers here, so we know what length of text to expect. Could be all 199.9 2999 1999 to indicate they are "fake".